members/Nicolas_Tschumy.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Nicolas Tschumy
+---
+
+Status: Phd student
+
+Expiration date: 2019.01.01
+
+Validity: <font color="red"> Not valid</font> 
```

Comment:
The status should be valid. Looks like Nicolas Tschumy paid:

16/12/2018 22:28:41 | Nicolas |   | Tschumy | Phd student |   | https://drive.google.com/open?id=1r75LNFhguWTIm174SHcOKRU880oXxjh_ | nicolas.tschumy@unil.ch | TRUE | TRUE |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

Do you have an idea why it's not vaild?

Reply:
This is totally normal, I added him to the list of people who paid after having run the script